pkgs/applications/audio/youtube-music/default.nix

```diff
@@ -11,13 +11,16 @@ let
 
   appimageContents = appimageTools.extract { inherit pname version src; };
 in
-appimageTools.wrapType2 rec {
+(appimageTools.wrapType2 rec {
   inherit pname version src;
   extraPkgs = pkgs: (appimageTools.defaultFhsEnvArgs.multiPkgs pkgs)
     ++ [ pkgs.libappindicator ];
 
   extraInstallCommands = ''
     mv $out/bin/{${pname}-${version},${pname}}
+    wrapProgram "$out/bin/${pname}" \
```

Comment:
```suggestion
    wrapProgram "$out/bin/youtube-music" \
```

Reply:
The `mv` command in the line above uses `pname`. Why the inconsistency?

Reply:
hmmm, you're right. Then we might as well keep it as is but generally we try to avoid using pname because the binary does not need to match and pname is sometimes changed by wrappers which break the assumption.